test/terraform_aws_cloudtrail_alarms_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+
+)
+
+func TestTerraformAwsCloudTrailAlarms(t *testing.T) {
+	awsRegion := "us-west-2"
+
+	t.Parallel()
+
+	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", ".")
+	alarm_sns_topic_arn := "arn:aws:cloudwatch:us-west-2:123456789012:alarm:myCloudWatchAlarm-CPUAlarm-UXMMZK36R55Z"
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+
+		// The path to where our Terraform code is located
+		TerraformDir: tempTestFolder,
+
+		// Variables to pass to our Terraform code using -var options
+		Vars: map[string]interface{}{
+			"alarm_sns_topic_arn": alarm_sns_topic_arn,
+			"cloudtrail_log_group_name": "test",
```

Comment:
This name will likely need to be randomly generated. This is an example of how to do that https://github.com/trussworks/terraform-aws-s3-private-bucket/blob/master/test/terraform_aws_s3_private_bucket_test.go#L21